repos/system_upgrade/common/actors/initramfs/upgradeinitramfsgenerator/libraries/upgradeinitramfsgenerator.py

```diff
@@ -21,6 +22,45 @@
 DRACUT_DIR = '/dracut'
 
 
+def _get_target_kernel_version(context):
+    """
+    Get the version of the most recent kernel version within the container.
+    """
+
+    kernel_version = None
+    try:
+        results = context.call(['rpm', '-qa', 'kernel'], split=True)
+
+        versions = map(lambda _: _.replace('kernel-', ''), results['stdout'])
```

Comment:
Functional style is really cool, but in his particular situation it's not necessary so it kinda contradicts our style guidelines https://leapp.readthedocs.io/en/latest/python-coding-guidelines.html#avoid-map-reduce-and-too-complex-comprehensions

Reply:
Thanks, forgot about that. Fixed!